src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc

```diff
@@ -1,2 +1,192 @@
+
+
+== Client part
+
+Finally, we just need to call the cloud functions to create a user account, log in and log out from the client part.
+You can replace the `front/index.js` file by the following content :
+
+.front/index.js
+[source, javascript, linenums]
+----
+// Create the ZetaPush API
+const client = new ZetaPushClient.SmartClient();
+const api = client.createProxyTaskService({ <1>
+  namespace: 'user'
+});
+
+// User properties
+const user = {
+  login: 'test',
+  password: 'password',
+  email: 'test-zetapush@yopmail.com'
+};
+
+async function signUpUser() { <2>
+  // First connection as anonymous user to allow us to create a user
+  await client.connect();
+
+  // Creation of the user account
+  try {
+    await api.signup({
+      credentials: {
+        login: user.login,
+        password: user.password
+      },
+      profile: {
+        email: user.email
+      }
+    });
+    console.log('User account created');
+  } catch (err) {
+    console.error('Failed to create user account : ', err);
+  }
+}
+
+/**
+ * Connection of a user
+ */
+async function connectUser() { <3>
+  await client.setCredentials({ login: user.login, password: user.password });
+  client
+    .connect()
+    .then(() => {
+      console.log('User connected');
+    })
+    .catch(err => {
+      console.error('Failed to connect user : ', err);
+    });
+}
+
+/**
+ * Disconnection of a user
+ */
+async function disconnectUser() { <4>
+  await client.disconnect();
+  console.log('User disconnected');
+}
+----
+<1> We created a ZetaPush client and his API to call our cloud functions (We use the namespace 'user' to call the **Standard User Workflow**).
+<2> First function to create an account of the ZetaPush platform for this application.
+<3> Second function to connect a user.
+<4> Third function to disconnect a user.
+
+Here is a very basic HTML page to see the result in the console :
+
+.front/index.html
+[source, html, linenums]
+----
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Standard User Workflow</title>
+</head>
+
+<body>
+  <button onclick="signUpUser()">Sign up</button>
+  <button onclick="connectUser()">Connect</button>
+
+  <script src="https://unpkg.com/@zetapush/client"></script>
+  <script src="./index.js"></script>
+</body>
+
+</html>
+----
+
+Run the following command to run your application in local and serve the front part :
+
+.Run the application
+[source, console]
+----
+$ npm run start -- --serve-front
+----
+
+NOTE: You can see a better example of the **Standard User Worflow** on https://github.com/zetapush/zetapush-examples/tree/master/standard-user-workflow[GitHub ZetaPush Examples].
+
+Now you can see the link:#+_standard_user_workflow+[advanced configuration], learn how to extend this cloud service via the link:#+_standard_user_workflow_2+[extensions] or link:#+_standard_user_workflow_3+[adding your own implementations].
```

Comment:
Try to use links as a normal part of the sentence:

You can now learn link:#+_standard_user_workflow+[how to deeply configure the Standard User Workflow], learn link:#+_standard_user_workflow_2+[how to extend this cloud service] and learn link:#+_standard_user_workflow_2+[how to use your custom implementations istead of built-in implemenations].

Why the names are numbered ?
+_standard_user_workflow+
+_standard_user_workflow_2+
+_standard_user_workflow_3+

It will be hard to maintain and to know what it references in 6 months